.travis.yml

```diff
@@ -10,12 +15,14 @@ env:
     - MOZDOWNLOADPARAMS='-t release -v latest'
     - MOZDOWNLOADPARAMS='-t release -v latest-esr'
 
-before_install:
-  - sudo apt-get update -qq
+addons:
+  apt:
+    packages:
+    - ant
+    - python-pip
 
 install:
-  - sudo apt-get install -y ant python-pip
-  - sudo pip install mozdownload==1.12
+  - pip install --user mozdownload==1.12
```

Comment:
Maybe make it even without a fixed version so we always use the latest version for testing.